.gitignore

```diff
@@ -0,0 +1,5 @@
+.gitcredentials
+apikey
+kubeconfig*
+*.yaml
```

Comment:
Careful with ignoring all `.yaml` and `.json` files by default. If you want to ignore test outputs, put them in a temp directory and ignore that. This will prevent headaches later when you have unit test corpus.